src/RedshiftAutoEncrypterUtility/README.md

```diff
@@ -0,0 +1,80 @@
+# Redshift Auto Encrypter Utility
+
+As you might be aware, in place or direct conversion from an unencrypted to encrypted cluster is not possible in Redshift. To migrate from an unencrypted Redshift cluster to an encrypted Redshift cluster, you first need to unload your data from the existing source cluster. Then you reload the data in a new target cluster with the chosen encryption setting. Post migration, you also need to [rename](https://docs.aws.amazon.com/redshift/latest/mgmt/working-with-clusters.html#rs-mgmt-rename-cluster) the cluster. For more information about launching an encrypted cluster, see [Amazon Redshift Database Encryption](https://docs.aws.amazon.com/redshift/latest/mgmt/working-with-db-encryption.html). As we can see, there are a couple of intermediate steps involved in the whole migration process. This is an utility which tries to automate this intermediate steps and thus help for seamless Redshift migration from unencrypted to encrypted cluster.
+
+## Running the Script
+
+You can follow the below instructions on you local linux machine but we recommend launching a Amazon linux EC2 Instance and attach a role to it with admin access in order to simplify the process of installing dependencies for the script. For instructions on how to do this, please refer below links:
+
+https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launching-instance.html<br>
+https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html
+
+1. Download the 'migrate-production.py' script from the GitHub repository to S3 or to an EC2 instance from where you can access the cluster.
+
+For example, you can download the script by using 'wget' command as given below:
+
+```bash
+wget https://raw.githubusercontent.com/suvenduk/amazon-redshift-utils/master/src/RedshiftAutoEncrypterUtility/migrate-production.py
```

Comment:
this path will need updating to be correct for the merged target.

Reply:
Maybe we can store the script in a S3 bucket